### ISSUE TITLE
Cannot configure DB used on cli commands

### ISSUE THREAD

**Issue by slamora** [eReuse/devicehub-teal#45]
When you run cli commands, e.g. to create initial common data and first inventory:
```bash
export dhi=dhtest;
dh inv add --common --tag-token 141b83d6-293f-4253-b22a-64edf408b573
```
You cannot configure the database that is used, as pointed on the **TODO** comment of [`ereuse_devicehub/cli.py`](https://github.com/eReuse/devicehub-teal/blob/testing/ereuse_devicehub/cli.py#L12) file.

## Steps to reproduce the issue
1. Create a custom database (different from the one hardcoded on [`ereuse_devicehub/config.py`](https://github.com/eReuse/devicehub-teal/blob/testing/ereuse_devicehub/config.py#L30).
2. Try to execute a cli command:
   `dh user add user@example.org`
3. Command will apply changes to default hardcoded database.

## Expected behaviour
Be able to use a custom database on cli.

**Comment by JNadeu** [comment]
For now, would it not be easier and more practical to manually modify SQLALCHEMY_DATABASE_URI on [`ereuse_devicehub/config.py`](https://github.com/eReuse/devicehub-teal/blob/testing/ereuse_devicehub/config.py#L30) ?

Because this is only done once when creating the DB. I don't know, it seems to me much more critical the issue of test suite or permissions.

@slamora If I understood correctly, the issue is to be able to create several DBs on the same server with several instances of DH using only the CLI. What do we gain with this?

**Comment by fedjo** [comment]
Hello @slamora 

Not sure if I can reproduce your error.

I edited  [`ereuse_devicehub/config.py`](https://github.com/eReuse/devicehub-teal/blob/testing/ereuse_devicehub/config.py#L30) and I used the following value
```
SQLALCHEMY_DATABASE_URI = 'postgresql://dhub:ereuse@localhost/devicehub1'
```

I then executed script to create the new database `devicehub1`
```
sh setup_postgres.sh
```
Then I execute
```
export dhi=dbtest;  dh inv add --common --name dbtest 
dh user add yiorgos@example.org
```

And a new user is created on db `devicehub1`.

I suppose I am doing sth different from your setup. Can you please expain (or post) you try to setup the new configuration?

Thanks

**Comment by slamora** [comment]
Of course, if you edit [`ereuse_devicehub/config.py`](https://github.com/eReuse/devicehub-teal/blob/testing/ereuse_devicehub/config.py#L30) you can customize the DATABASE config; but I believe that a good practice is to decouple the code from the deployment configuration.

So, the my proposal is to find a way to decouple code from configuration in a similar way that flask allows to use an app to define the configuration.

**Comment by cayop** [comment]
This was fixed when we added decouple: [ec671312db10ed84070ea212e2fcbf6acd4b25c6]
